azerothcore/azerothcore-wotlk
# fix(DB/pool): Remove redundant Meshlok creature pool

Pool id 1211 is redundant and it needlessly reduces the spawn chance of Meshlok.
Rare creature spawn chance should be handled exclusively by SAI

Courtesy of @T1ti 



## Changes Proposed:
-  remove pool 1211

## Issues Addressed:

- Closes #6745
- Closes https://github.com/chromiecraft/chromiecraft/issues/1098

## SOURCE:


## Tests Performed:

- sql applies cleanly. Not tested in-game

## How to Test the Changes:


1. make sure sql applies cleanly
2. make sure worldserver doesn't throw warnings and stuff at you

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
